erigon-lib/diagnostics/stages.go

```diff
@@ -52,3 +67,45 @@ func (d *DiagnosticClient) runCurrentSyncStageListener(rootCtx context.Context)
 		}
 	}()
 }
+
+func ReadSyncStagesInfo(db kv.RoDB) (info SyncStages) {
+	stageesList := ReadStagesList(db)
+	currentStage := ReadCurrentStage(db)
+
+	return SyncStages{
+		StagesList:   stageesList,
+		CurrentStage: currentStage,
+	}
+}
+
+func ReadStagesList(db kv.RoDB) []string {
+	data := ReadDataFromTable(db, kv.DiagSyncStages, StagesListKey)
+	var info []string
+	err := json.Unmarshal(data, &info)
+
+	if err != nil {
+		return []string{}
```

Comment:
is it ok to just omit error? data is stored in db but each time it could be just empty string or 0 without knowing something wrong with encoding - so maybe add error log entry when error happens and omitted?